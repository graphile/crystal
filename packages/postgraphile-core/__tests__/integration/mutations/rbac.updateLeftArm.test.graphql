#> rbac: true
#> options: { ignoreRBAC: false }

mutation {
  updateLeftArmById(input:{
    id: 47
    leftArmPatch:{
      mood: "jovial"
    }
  }) {
    leftArm {
      nodeId
      id
      personId
      lengthInMetres
      mood
    }
  }
}
