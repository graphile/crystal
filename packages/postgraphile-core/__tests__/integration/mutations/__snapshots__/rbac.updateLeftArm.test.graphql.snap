// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "updateLeftArmById": Object {
      "leftArm": Object {
        "id": 47,
        "lengthInMetres": 0.65,
        "mood": "jovial",
        "nodeId": "WyJsZWZ0X2FybXMiLDQ3XQ==",
        "personId": 5,
      },
    },
  },
}
`;
